src/client-ducky/src/components/Detail.js

```diff
@@ -10,6 +10,7 @@ const Detail = ({spot_id, isAuthenticated, token}) => {
 
     function getSpot(pk) {
         api.get(`parking-spots/${pk}/`).then(res => {
+            debugger;
```

Comment:
next time just look for debuggers before the pr, but we'll just remove this one later

Reply:
kk